packages/common/amplify_db_common/README.md

```diff
@@ -1,15 +1,15 @@
-## amplify_db_common
+# amplify_db_common
 
```

Comment:
Can we at least mention this is temporary here to keep track.  In case the team forgets this is a temporary fix.

Reply:
This top level Flutter library will replace our import of sqlite3_flutter_libs in the short term until we remove our dependency on iOS DataStore.  When that happens we will switch to using sqlite3_flutter_libs and the amplify_db_common_dart package instead.

Reply:
I'd rather not publish something saying it's a temporary fix. However, this package will be needed even once we decide to use the flutter_libs package. It incorporates path_provider into the `connect` function.